packages/components/src/TabBar/TabBar.scss

```diff
@@ -10,13 +10,62 @@
 	background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M8 13.6l-8-8L2.6 3 8 8.4 13.4 3 16 5.6z" fill="#{hex-to-url($color)}"/></svg>');
 }
 
+$underline-height: 2px;
+
 .tc-tab-bar {
 	white-space: nowrap;
 	overflow: hidden;
 
 	& &-item {
 		float: none;
 		display: inline-block;
+
+		&-icon {
+			margin-right: $padding-smaller;
+			width: $svg-sm-size;
+			height: $svg-sm-size;
+			font-size: $svg-sm-size;
+		}
+
+		button {
+			padding: $padding-smaller $padding-small;
+			padding-bottom: calc(#{$padding-smaller} + #{$underline-height});
+			color: $black;
+
+			max-width: 200px;
+			white-space: nowrap;
+			overflow: hidden;
+			text-overflow: ellipsis;
+
+			&:hover {
```

Comment:
and `&:active`, no?

Reply:
active is already has this color. It's defined on line 49.
By the way it won't work here, such as button doesn't have active class. But it will work with &[aria-selected='true']